web/store/index.js

```diff
@@ -70,8 +74,17 @@ export default new Vuex.Store({
     },
     // LOCATION
     locationFeatureType(state) {
-      const { location } = state;
-      return getLocationFeatureType(location);
+      const { locations } = state;
```

Comment:
we have a couple of places (e.g. map, Request Study) where we offer a computed property `location`, which is really just the first location (or `null` if the selection is empty) - this allows us to migrate gradually without breaking existing flows.

can we move that computed property in here as a getter, and add a TODO on it to remove once we're much closer to finishing?